10-std.tex

```diff
@@ -539,3 +539,145 @@ \section{Input/Output}
+		\end{description}
+		
+	\item[Neko:]
+		\begin{description}
+			\item[\expr{HttpConnection.urlConnect(url:String)}]  
+				Will work as the asynchronous version but in synchronous mode.
+			\item[\expr{SocketConnection.create(...)}]  
+				Allows real-time communications with a Flash client which is using an \type{XMLSocket} to connect to the server.
+		\end{description}
+\end{description}
+
+\paragraph{Remoting context}
+
+Before we actually communicate between platforms, a remoting context have to be defined. This is a shared API that can be called on the connection at the client code.
+
+This server code example creates and shares an API:
+\begin{lstlisting}
+class Server {
+	function new() { }
+	function foo(x, y) { return x + y; }
+
+	static function main() {
+		var ctx = new haxe.remoting.Context();
+		ctx.addObject("Server", new Server());
+		
+		if(haxe.remoting.HttpConnection.handleRequest(ctx))
+		{
+			return;
+		}
+		
+		// handle normal request
+		trace("This is a remoting server !");
+	} 
+}
+\end{lstlisting}
+
+\paragraph{Using the connection}
+
+Using a connection is pretty convenient. Once the connection is obtained, use classic dot-access to evaluate a path and then use \expr{call()} to call the method in the remoting context and get the result.
+The asynchronous connection takes an additional function parameter that will be called when the result is available.
+
+This client code example connects to the server remoting context, and calls a function \expr{foo()} on its API.
+\begin{lstlisting}
+class Client {
+  static function main() {
+    var cnx = haxe.remoting.HttpAsyncConnection.urlConnect("http://localhost/");
+    cnx.setErrorHandler( function(err) trace('Error: $err'); } );
+    cnx.Server.foo.call([1,2], function(data) trace('Result: $data'););
+  }
+}
+\end{lstlisting}
+
+To make this work for Neko target, setup a Neko Web Server, point the url in the Client to \ic{"http://localhost2000/remoting.n"} and compile the Server using \ic{-main Server -neko remoting.n}.
+
+\paragraph{Error handling}
+
+\begin{itemize}
+	\item When an error occurs in a asynchronous call, the error handler is called as seen in this example above.
+	\item When an error occurs in a synchronous call, an exception is raised on the caller-side, as if we were calling a local method.
+\end{itemize}
+
+\paragraph{Data serialization}
+
+Haxe Remoting can send a lot of different kinds of data. See \tref{Serialization}{std-serialization}.
+
+\subsection{Implementation details}
+\label{std-remoting-implementation-details}
+
+\paragraph{Javascript security specifics}
+
+The html-page wrapping the js client must be served from the same domain as where the server is running. The same-origin policy restricts how a document or script loaded from one origin can interact with a resource from another origin. Same-origin Policy is used as a means to prevent some of the Cross-site Request Forgery attacks.
```

Comment:
"as the one where the server is running...". The same-origin polycy isused (decide whether to capitalize or not, use an article (the) in both cases)